apps/web/src/app/(dashboard)/documents/[id]/document-page-view.tsx

```diff
@@ -162,13 +171,17 @@ export const DocumentPageView = async ({ params, team }: DocumentPageViewProps)
           </CardContent>
         </Card>
 
-        {document.status === DocumentStatus.PENDING && (
+        {document.status === DocumentStatus.COMPLETED && (
```

Comment:
We don't need to display fields for `COMPLETED` documents since they're already imprinted onto the PDF itself